plugins/flatpak/gs-flatpak-app.h

```diff
@@ -72,6 +72,9 @@ void			 gs_flatpak_app_set_file_kind		(GsApp		*app,
 								 GsFlatpakAppFileKind	file_kind);
 void			 gs_flatpak_app_set_runtime_repo	(GsApp		*app,
 								 GsApp		*runtime_repo);
+void			 gs_flatpak_app_set_main_app		(GsApp		*app,
+								 const gchar	*main_app_ref);
+const gchar		*gs_flatpak_app_get_main_app		(GsApp		*app);
```

Comment:
I find it a bit strange that this method is called `_get_main_app()` and it returns a `const char *` instead of a GsApp. IMHO, it should either return a `GsApp` or be renamed to something like `_get_main_app_name()`.

Reply:
I agree. I will rename it as `gs_flatpak_app_get_main_app_ref`.